services/cd-service/pkg/repository/transformer_db_test.go

```diff
@@ -1297,7 +1472,7 @@ func TestMinorFlag(t *testing.T) {
 						return err
 					}
 					if !release.Metadata.IsMinor {
-						t.Errorf("Expected release %d to be minor but its major", minorVersion)
+						t.Errorf("Expected release %d.%d to be minor but its major", *minorVersion.Version, minorVersion.Revision)
```

Comment:
nitpick:
if releaseNumbers implemented a String function, printing would be a bit more convenient
```suggestion
						t.Errorf("Expected release %v to be minor but its major", minorVersion)
```

See https://pkg.go.dev/fmt#Stringer